docs/book/src/index.md

```diff
@@ -30,7 +30,7 @@ Whilst Rust is a great systems programming language (and Sway itself is written
 
 Rust shines because it can use zero-cost abstractions and its sophisticated borrow-checker memory model to achieve impressive runtime performance for complex programs without a garbage collector.
 
-On a blockchain, cost of execution and deployment is the scarce resource.
+On a blockchain, cost of execution and deployment is scarce.
 Memory usage is low and execution time is short.
```

Comment:
```suggestion
Memory is an expensive resource.
```

Reply:
You're losing key information and this isn't what I mean here.

Reply:
Updated